upload.py

```diff
@@ -34,6 +34,9 @@ def main():
     meta_args = ()
     if args.meta_path:
         meta_args = ('--module', args.meta_path)
+    elif args.name:
+        meta_args = ('--name', args.name)
+
```

Comment:
This didn't seem to be populated before

Reply:
for clarity -- you're intentionally removing the `org-id` arg from the `update` command?

Reply:
Yes, the org-id and namespace arguments don't make much sense for the update command because the update command requires a meta.json (which includes the module_id (which includes namespace and orgid)). 

Back when `name` existed, it kind of vaguely made sense, but it really was just complication because it is possible to write it that way, not because it was actually good behavior.